src/modules/RA/DataLayer/components/DataLayerTabbedForm.js

```diff
@@ -53,276 +52,300 @@ const LazyFormTab = ({ hidden, ...props }) => (
   hidden ? null : <FormTab {...props} />
 );
 
+const DisplayDataTableField = () => {
+  const form = useForm();
+
+  return (
+    <BooleanInput
+      source="table_enable"
+      label="datalayer.form.allow-display-data-table"
+      onChange={value => {
+        if (!value) return;
+        form.change('table_export_enable', false);
+      }}
+    />
+  );
+};
+
+const SourceField = ({ formData, dataProvider }) => {
+  const form = useForm();
+  return (
+    <>
```

Comment:
Do we need this wrapper ?

Reply:
Not anymore *(as `FieldUpdater is now inside a hidden tab)*